### ISSUE TITLE
Uncaught Exception when use %Z or %z date formatter in OFDate method localDateStringWithFormat:

### ISSUE THREAD

**Issue by GunGraveKoga** [Midar/objfw#16]
Like seid in http://www.cplusplus.com/reference/ctime/strftime/ 
%Z	|| Timezone name or abbreviation If timezone cannot be determined, no characters || CDT

But in my application I get exception like this:
[2015-10-24 22:11:07.729 fcgizonecreator.exe(14424)] Dispatch task in thread 14428
[2015-10-24 22:11:07.731 fcgizonecreator.exe(14424)] Application retain count^ 1
[2015-10-24 22:11:07.731 fcgizonecreator.exe(14424)] [14408]Request:test3.php recived on Server: webzonemanager <localhost:9000>
[2015-10-24 22:11:07.732 fcgizonecreator.exe(14424)] An encoding is invalid!
Try load Dynamorio DLL's...
loaded
[2015-10-24 22:11:07.950 fcgizonecreator.exe(14424)] 0 libobjfw.dll 6F1787D7 -[OFException init] +0x000F8740 [c:\Proj\objfw-master\src\exceptions/OFException.m : 240]
[2015-10-24 22:11:07.951 fcgizonecreator.exe(14424)] 1 libobjfw.dll 6F17870B +[OFException exception] +0x000F8690 [c:\Proj\objfw-master\src\exceptions/OFException.m : 234]
[2015-10-24 22:11:07.951 fcgizonecreator.exe(14424)] 2 libobjfw.dll 6F150BC4 -[OFString_UTF8 OF_initWithUTF8String_length_storage_] +0x000D09D0 [c:\Proj\objfw-master\src/OFString_UTF8.m : 195]
[2015-10-24 22:11:07.952 fcgizonecreator.exe(14424)] 3 libobjfw.dll 6F0DF59A -[OFString_placeholder initWithUTF8String_] +0x0005F4A0 [c:\Proj\objfw-master\src/OFString.m : 317]
[2015-10-24 22:11:07.952 fcgizonecreator.exe(14424)] 4 libobjfw.dll 6F0E09AF +[OFString stringWithUTF8String_] +0x00060920 [c:\Proj\objfw-master\src/OFString.m : 560]
[2015-10-24 22:11:07.952 fcgizonecreator.exe(14424)] 5 libobjfw.dll 6F09B11B -[OFDate localDateStringWithFormat_] +0x0001ACD0 [c:\Proj\objfw-master\src/OFDate.m : 555]
[2015-10-24 22:11:07.952 fcgizonecreator.exe(14424)] 6 fcgizonecreator.exe 00401F20 __50-[OFFCGIServer server:didReceiveRequest:response:]_block_invoke +0x00001CD0 [./fcgizonecreator.m : 85]
[2015-10-24 22:11:07.953 fcgizonecreator.exe(14424)] 7 libdispatch.dll 705822ED _dispatch_call_block_and_release +0x000022D0 [c:/Proj/xdispatch-master/libdispatch/src/blocks.c : 45]
.......

**Comment by Midar** [comment]
Quoting from the documentation:

```
 * @warning The format is currently limited to the following format specifiers:
 *          %%d, %%e, %%H, %%m, %%M, %%S, %%y, %%Y, %%, %%n and %%t.
```

It's therefore not a bug, but just not supported, which is why you get the exception. Do you want to change this to a feature request?

**Comment by GunGraveKoga** [comment]
Yes, if it's possible
As I see localDateStringWithFormat: uses strftime() C stdlib function, MSDN says that %Z or %z parameters are supported

**Comment by Midar** [comment]
Oops, sorry, I was actually looking at strptime, not strftime. Yes, in that case, what MSDN says should actually be true on Windows. I'll look into this.

**Comment by Midar** [comment]
Sorry, I should have had a more thorough look at your stack trace. This is what happens when you only quickly look at the bug report in the middle of something ;).

Since the exception is actually coming from OFString_UTF8, I suppose the problem is that strftime on Windows returns something that is not valid UTF-8. I'll see if there's a better way to do this on Windows.

**Comment by Midar** [comment]
Turns out my assumption was right: strftime on Windows does indeed return invalid UTF-8, for example "Mitteleuropäische Zeit", where the ä is not in UTF-8. I fixed this now in master and the 0.8 branch by using wcsftime on Windows. However, this is still a little bit weird, as %z is defined differently by C99 than it is on Windows (C99 says it's the ISO 8601 timezone offset, whereas Windows has the full time zone name).
